BREAKING.md

```diff
@@ -19,8 +19,10 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 
 ## 2.0.0-internal.2.4.0 Upcoming changes
 - [Deprecate `ensureContainerConnected()` in `@fluidframework/test-utils`](#deprecate-ensurecontainerconnected-in-fluidframeworktest-utils)
+- [Deprecate `AzureFunctionTokenProvider` in `@fluidframework/azure-clients`](#deprecate-`AzureFunctionTokenProvider`-in-fluidframeworkazure-client)
```

Comment:
This is Azure release so this would not end up being internal-2.4.0 change. We need to sort out breaking communication for azure. cc @scottn12 @tylerbutler